src/meshlabplugins/filter_mutualglobal/alignGlobal.h

```diff
@@ -30,7 +30,7 @@ Revision 1.1  2006/09/25 09:24:39  e_cerisoli
 add sampleplugins
 
 ****************************************************************************/
-
+#pragma once

```

Comment:
Beware that `#pragma once` , although supported by almost every modern compiler, is not C++ standard.

Reply:
Yes, I do know that unfortunately. (What compiler doesn't support it?) I can change this to a normal include if that better fits project rules.